<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureDevicePropertyClass</class>
 <widget class="QMainWindow" name="CaptureDevicePropertyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaptureDeviceProperty</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>select capture device, and press &quot;Show Property&quot; button.</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="listDevice"/>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="buttonReload">
            <property name="text">
             <string>Reload</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonShowProperty">
            <property name="text">
             <string>Show Property</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="capturedeviceproperty.qrc"/>
 </resources>
 <connections/>
</ui>
